### PR TITLE
Add Aerospike metrics receiver

### PR DESCRIPTION
**Pull request by antonblock**
Add metrics receiver for Aerospike

### CODE REVIEW

File: integration_test/third_party_apps_data/applications/aerospike/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+app_url: "https://aerospike.com/"
+short_name: Aerospike
+long_name: Aerospike
+logo_path: ""
```
Comment (sophieyfang) [inline integration_test/third_party_apps_data/applications/aerospike/metadata.yaml:4]: I think Mary @mtabasko will know what to set for this field :)

**Review by ridwanmsharif — COMMENTED** [review]
Looks good overall, left a few comments but is close.

File: integration_test/third_party_apps_data/applications/aerospike/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.node.memory.free
+    value_type: INT64
+configuration_options:
+  metrics:
+  - type: aerospike
+    fields:
+    - name: type
+      default: null
+      description: This value must be aerospike.
```
Comment (mtabasko) [inline integration_test/third_party_apps_data/applications/aerospike/metadata.yaml:81]: `aerospike`

File: integration_test/third_party_apps_data/applications/aerospike/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    - name: collection_interval
+      default: 60s
+      description: A time.Duration value, such as 30s or 5m.
```
Comment (mtabasko) [inline integration_test/third_party_apps_data/applications/aerospike/metadata.yaml:84]: A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`.

File: integration_test/third_party_apps_data/applications/aerospike/metadata.yaml (hunk 1 above)
Comment (mtabasko) [reply integration_test/third_party_apps_data/applications/aerospike/metadata.yaml:4]: Let's go with "/stackdriver/images/integrations/aerospike.png"

(I don't have anything there yet, but that's where Evan's team decided they want logos, so we'll go with that.)

File: integration_test/third_party_apps_data/applications/aerospike/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    - name: endpoint
+      default: localhost:3000
+      description: The JMX Service URL or host and port used to construct the service URL. This value must be in the form of service:jmx:<protocol>:<sap> or host:port. Values in host:port form are used to create a service URL of service:jmx:rmi:///jndi/rmi://<host>:<port>/jmxrmi.
```
Comment (mtabasko) [inline integration_test/third_party_apps_data/applications/aerospike/metadata.yaml:87]: The [JMX Service URL](https://docs.oracle.com/javase/8/docs/api/javax/management/remote/JMXServiceURL.html) or host and port used to construct the service URL. This value must be in the form of `host:port`, which is used to create a service URL of the form `service:jmx:remote+http://<host>:<port>/jmxrmi`.

**Review by mtabasko — CHANGES_REQUESTED** [review]

File: integration_test/third_party_apps_data/applications/aerospike/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  - kind: GAUGE
+    labels:
+      node_name: .*
+      namespace_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.namespace.memory.free
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      component: data|index|set_index|secondary_index
```
[the remark this answers is not in the document]
Comment (antonblock) [reply integration_test/third_party_apps_data/applications/aerospike/metadata.yaml:38]: Scope doc needs updating, just updated it

File: integration_test/third_party_apps_data/applications/aerospike/metadata.yaml (hunk 1 above)
Comment (antonblock) [reply integration_test/third_party_apps_data/applications/aerospike/metadata.yaml]: Added this

**Comment by antonblock** [comment]
@ridwanmsharif I believe all the feedback is addressed here. `Ops Agent integration test (Bullseye)` is failing syslog tests, so I'm pretty sure it's unrelated to this PR:
```
=== CONT  TestSyslogTCP/debian-11
    ops_agent_test.go:697: Error writing dummy log line: Command failed: logger -n 0.0.0.0 --tcp --port=5140 -- abc test pattern xyz
        Command failed: [ssh test_user@10.128.0.190 -oIdentityFile=/tmp/ssh_keys4150816588/gce_testing_key -oConnectTimeout=120 -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -oLogLevel=ERROR logger -n 0.0.0.0 --tcp --port=5140 -- abc test pattern xyz]
        exit status 1
        stdout+stderr: logger: failed to connect to 0.0.0.0 port 5140
```

**Review by ridwanmsharif — APPROVED** [review]

**Comment by ridwanmsharif** [comment]
Will merge if/when @mtabasko green lights the docs changes in the `metadata.yaml` file.

**Pull request merged**